.github/workflows/pull-request-account-compression.yml

```diff
@@ -8,6 +8,7 @@ on:
       - "ci/*-version.sh"
       - "ci/install-anchor.sh"
       - ".github/workflows/pull-request-account-compression.yml"
+      - "!account-compression/sdk/**"
```

Comment:
what does this do & why add?

Reply:
The `!` means to not run if there's a change in one of those files.

The idea is to *not* run this workflow if there's *only* a change to the JS side, since `pull-request-js.yml` gets run if there's only a change on the JS side. But if there's a change in the program, you *do* want to run this workflow.

Reply:
But if there's a change in the SDK the tests should run. SDK can change test behavior

Reply:
Yep, the SDK tests are run by `pull-request-js.yml` if there's a change there